packages/components/pie-icon-button/package.json

```diff
@@ -25,13 +25,11 @@
   },
   "author": "Just Eat Takeaway.com - Design System Team",
   "license": "Apache-2.0",
-  "devDependencies": {
+  "dependencies": {
     "@justeattakeaway/pie-icons-webc": "0.11.1",
+    "@justeattakeaway/pie-spinner": "0.1.0",
     "@justeattakeaway/pie-webc-core": "0.11.0"
   },
-  "peerDependencies": {
-    "@justeat/pie-design-tokens": "^5.8.2"
```

Comment:
I think we'll likely need to replace this with a `peerDependency` on `pie-css` (as that import contains the design tokens now).